installing/acs-installation-platforms.adoc

```diff
@@ -6,7 +6,7 @@ include::modules/common-attributes.adoc[]
 toc::[]
 
 [role="_abstract"]
-{rh-rhacs-first} and {rh-rhacscs-first} are supported on {ocp} and Kubernetes platforms.
+{rh-rhacs-first} is supported on {ocp} and Kubernetes platforms. For more details about supported self-managed and managed platforms, see link:https://access.redhat.com/node/5822721[{product-title} Support Policy].
```

Comment:
```suggestion
{rh-rhacs-first} is supported on {ocp} and Kubernetes platforms. For more information about supported self-managed and managed platforms, see link:https://access.redhat.com/node/5822721[{product-title} Support Policy].
```
I believe we standardize on `information` [1]
[1] https://redhat-documentation.github.io/supplementary-style-guide/#cross-references